engines/bastion/app/assets/javascripts/bastion/components/views/bst-edit.html

```diff
@@ -14,6 +14,8 @@
     <div ng-switch-when="addRemove">
       <div ng-include="'components/views/bst-edit-add-remove-cancel.html'"></div>
     </div>
+    <div ng-switch-when="none">
```

Comment:
Interesting. Why was this needed ?

Reply:
I'm using the new `bst-checkbox-list` in the New Repository form, where the Save button is at the bottom of the form.  I didn't want it to show its own 'save' and 'cancel' buttons here, when there is another one at the bottom of the form.

Reply:
I suppose this div could be eliminated, but I think it's better to be explicit and cover all `buttonConfig` options.